app/helpers/application_helper.rb

```diff
@@ -120,6 +120,10 @@ def check_icon
     inline_svg_tag 'check.svg'
   end
 
+  def mail_header_image(icon = 'email.png')
+    frontend_asset_url("images/mailer-new/heading/#{icon}")
```

Comment:
One delightful aspect of this is consolidating down the number of places "mailer-new" appears, so that we can migrate that to a more stable path.